kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -757,4 +731,38 @@ private List<String> getModemsPaths() {
         return modemsPath;
     }
 
+    private Optional<String> getModemPath(Device device) throws DBusException {
```

Comment:
Originally I thought about doing the same but, it's really hard to understand what path this method is going to return: is it the ModemManager Dbus path or the NetworkManager one? That's why I left the code as it was.